src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,23 @@
+package seedu.addressbook.commands;
+
+/**
+ * Clears the address book.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+    public static final String MESSAGE_USAGE = "Sorts address book by name. Persons with the same name" +
+            "will have their relative order preserved. \n"
```

Comment:
I think you got to add a space before `will` else it will appear weird on the help command =0.